xml/System.Text.Json/JsonDocument.xml

```diff
@@ -292,7 +292,7 @@ The <xref:T:System.ReadOnlyMemory`1> value may be used for the entire lifetime o
         <remarks>
           <format><![CDATA[
 
-If the <xref:System.Text.Json.Utf8JsonReader.TokenType> property of `reader` is <xref:System.Text.Json.JsonTokenType.PropertyName> or <xref:System.Text.Json.JsonTokenType.None?displayProperty=nameWithType>, the reader will advance by one call to <xref:System.Text.Json.Utf8JsonReader.Read?displayProperty=nameWithType> to determine the start of the value.
+If the <xref:System.Text.Json.Utf8JsonReader.TokenType> property of `reader` is <xref:System.Text.Json.JsonTokenType.PropertyName?displayProperty=nameWithType> or <xref:System.Text.Json.JsonTokenType.None?displayProperty=nameWithType>, the reader will advance by one call to <xref:System.Text.Json.Utf8JsonReader.Read?displayProperty=nameWithType> to determine the start of the value.
```

Comment:
Can you please explain what `displayProperty=nameWithType` do?

Reply:
Certainly, @ahsonkhan. For a type, it displays the fully qualified type name. For a member, it displays the type and member name. Generally, enumeration members should be displayed with both type and member name.